src/nodeTypesStore.js

```diff
@@ -27,6 +27,13 @@ export default new Vuex.Store({
           label: nodeType.label,
           bpmnType: nodeType.bpmnType,
           rank: nodeType.rank || BOTTOM,
+          items: nodeType.items && nodeType.items
+            .map(item => ({
+              type: item.id,
+              icon: item.icon,
+              label: item.label,
+              bpmnType: item.bpmnType,
+            })),
```

Comment:
why add a second array with the same data from `nodeTypes`

Reply:
It will be refactored, basically we only need the `type` property for the click & drop functionality and reuse the  `handleDrop` function of the Modeler component